tests/sibernetic/README.md

```diff
@@ -0,0 +1,20 @@
+## Test conversion of Sibernetic output to WCON
+
+Example output produced by Sibernetic & some Python scripts for converting this to WCON.
+
+**Work in progress!**
+
+Sibernetic original data is in [crawling_at_agar.txt](crawling_at_agar.txt).
```

Comment:
This is too much data I think to have as an example.  It should be long enough to see that it works, but that's about it.